code/obj/machinery/door/airlock.dm

```diff
@@ -1667,7 +1667,7 @@ About the new airlock wires panel:
 	return
 
 /obj/machinery/door/airlock/open()
-	if (src.welded || src.locked || src.operating == 1 || (!src.arePowerSystemsOn()) || (src.status & NOPOWER) || src.isWireCut(AIRLOCK_WIRE_OPEN_DOOR))
+	if (!src.density || src.welded || src.locked || src.operating == 1 || (!src.arePowerSystemsOn()) || (src.status & NOPOWER) || src.isWireCut(AIRLOCK_WIRE_OPEN_DOOR))
```

Comment:
I can't think of anything this would break, but I feel like there's a reason it isn't here. Better to explicitly put it in the wraith code just to be sure.

Reply:
First check of door/proc/open() is a density check too

Reply:
suuuure, probably fine